students/TimPauley/session04/config.py

```diff
@@ -0,0 +1,59 @@
+#Tim Pauley
+#Python 220, Assignment 04
+#Jan 29 2019
+
+#Assignment 04: 
+
+"""
+Test Configuration for Customer Model
+"""
+# I would normally do this in yaml or json, but did here for simplicity
+
+# prod
+# DATABASE = 'customers.db'
+
+##### CHANGE DATABASE VARIABLE, NOT TEST_DATABASE
+
+TEST_DATABASE = 'test.db'
+DATABASE = TEST_DATABASE
+
+### Should be in every config (normally would be separate config)
+etext = {
+    "not_found": "The customer with id: {} was not found",
+    "req_bool": "Status was {}, must be 'Active', 'Inactive', True or False",
+    "no_save": "We were unable to save customer with id {}, a required field was missing."
+}
```

Comment:
Yup, imagine if the code throwing the error and the test had access to the same error message. :P